roles/os_hardening/tasks/minimize_access.yml

```
@@ -80,3 +80,174 @@
     fstype: proc
     opts: '{{ proc_mnt_options }}'
     state: mounted
+
+- name: Mount /boot with hardened options
+  mount:
+    path: /boot
+    src: '{{ os_mnt_boot_src }}'
+    fstype: '{{ os_mnt_boot_filesystem }}'
+    opts: '{{ os_mnt_boot_options }}'
+    state: remounted
```

```suggestion
    state: '{{ item }}'
```

These changes are relevant for all tasks.